addons/crm/tests/test_performances.py

```diff
@@ -174,7 +174,7 @@ def test_assign_perf_populated(self):
                 lead.probability = (idx + 1) * 10 * ((int(lead.priority) + 1) / 2)
 
         with self.with_user('user_sales_manager'):
-            with self.assertQueryCount(user_sales_manager=6250):  # crm only: 6237
+            with self.assertQueryCount(user_sales_manager=6290):  # crm only: 6237
```

Comment:
Please run the test in local with just `crm` installed and update the comment to new value as well (if it changed?).

Reply:
@seb-odoo Done :+1: